src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,35 @@
+.reviews {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 16px;
+
+  &__star {
+    display: flex;
+    align-items: center;
+  }
+
+  &__star-item {
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-image: url(../images/star.svg);
+  }
+
+  &__star :last-child {
+    margin-right: 0;
+  }
+
+  &__star :nth-child(-n + 4) {
+    background-image: url(../images/star-active.svg);
+  }
```

Comment:
![image](https://github.com/mate-academy/layout_product-cards/assets/80700247/2de487ca-030a-435b-8b5d-f5ac4ecfa8ae)

You also have to add a modifier and if I change the value of the modifier in the `index.html` file, the number of yellow stars should also change